src/test/java/lotto/domain/LottoCheckerTest.java

```diff
@@ -0,0 +1,32 @@
+package lotto.domain;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class LottoCheckerTest {
+
+    @Test
+    @DisplayName("내가 산 복권과 당첨 복권이 3개 일치")
+    void countMatch() {
+        Lotto lotto = new Lotto(List.of(1, 2, 3, 4, 5, 6));
+        Lotto winning = new Lotto(List.of(1, 2, 3, 9, 10, 11));
+        assertThat(new LottoChecker().countMatch(lotto, winning)).isEqualTo(3);
+    }
+
+    @Test
+    void check() {
+        Lotto winning = new Lotto(List.of(1, 2, 3, 4, 5, 6));
+        List<Lotto> lottos = new ArrayList<>();
+        lottos.add(new Lotto(List.of(1, 2, 3, 10, 11, 12)));
+        lottos.add(new Lotto(List.of(1, 2, 11, 12, 13, 14)));
```

Comment:
```suggestion
        List<Lotto> lottos = List.of(new Lotto(1, 2, 3, 10, 11, 12), new Lotto(1, 2, 11, 12, 13, 14));
```
위와 같이 List.of를 사용하는 것을 추천
코드의 간결함 이외에도 이유가 있는데 왜 그런지는 AI를 통해 피드백을 받아본다.